recipes/ffmpeg/all/conanfile.py

```diff
@@ -333,7 +333,7 @@ def _configure_autotools(self):
             "--disable-cuda",  # FIXME: CUDA support
             "--disable-cuvid",  # FIXME: CUVID support
             # Licenses
-            opt_enable_disable("nonfree", self.options.with_libfdk_aac),
+            opt_enable_disable("nonfree", self.options.with_libfdk_aac or ( self.options.with_ssl and ( self.options.with_libx264 or self.options.with_libx265 or self.options.postproc ) ) ),
```

Comment:
condition is super-complicated, shouldn't it be just:
```
with_libfdk_aac or with_ssl or with_libx264 or with_libx265
```

Reply:
I agree the codition is super-complicated.

But I've put this because :
- `self.options.with_libx264 or self.options.with_libx265 or self.options.postproc` --> Enable GPL but not nonfree
https://github.com/conan-io/conan-center-index/blob/36ccc6fd53df01885560830447161ac1d7c419cb/recipes/ffmpeg/all/conanfile.py#L337
if we set `opt_enable_disable("nonfree", with_libfdk_aac or with_ssl or with_libx264 or with_libx265 or postproc ), `, nonfree will always be enable if GPL is enable

And we just want to enable `nonfree` if SSL and GPL are enable at the same time.

Reply:
maybe it can be simplified a bit, but okay, I got the idea